src/PKSim.Core/Model/ParameterRange.cs

```diff
@@ -32,15 +19,20 @@ public ParameterRange()
       public IDimension Dimension { get; set; }
       public Unit Unit { get; set; }
 
-      public bool IsConstant
+      public ParameterRange()
       {
-         get { return MinValue.HasValue && MaxValue.HasValue && MinValue.Value == MaxValue.Value; }
+         Rules = new BusinessRuleSet();
+         Rules.Add(AllRules.MinLessThanMax);
+         Rules.Add(AllRules.MaxGreaterThanMin);
+         Rules.Add(AllRules.MinGreaterThanDbMin);
+         Rules.Add(AllRules.MaxLessThanDbMax);
+         Rules.Add(AllRules.MinLessThanDbMax);
+         Rules.Add(AllRules.MaxGreaterThanDbMin);
```

Comment:
I think `MinLessThanMax `& `MinGreaterThanDbMin ` & `MaxLessThanDbMax `implies both `MaxGreaterThanDbMin`
and `MinLessThanDbMax`

Reply:
???

Reply:
This code has nothing to do with the PR BTW

Reply:
I did not see the implies. This might be correct but the rules should be symetrical so that it does not feel like something missing.

Reply:
Well, it's marked as new code by GitHub.

I  mean e.g. if both `MinLessThanMax ` and `MinGreaterThanDbMin ` rules are satisfied, than `MaxGreaterThanDbMin` is automatically satisfied, so why would we need this additional rule?

Same for  `MinLessThanDbMax`

Reply:
I think it's marked that way because the code moved.

Reply:
@Yuri05  I don't think this is true 
it could be that there is a DBMin but no min is defined by the user. In that case we need to check that the Max is greater than the DBMin
But at any rate, this code is old and was not changed (Aside from formatting probably)

Reply:
ok